src/hotspot/share/gc/shared/gc_globals.hpp

```diff
@@ -693,8 +693,12 @@
   product(uint, GCCardSizeInBytes, 512,                                     \
           "Card table entry size (in bytes) for card based collectors")     \
           range(128, NOT_LP64(512) LP64_ONLY(1024))                         \
-          constraint(GCCardSizeInBytesConstraintFunc,AtParse)
-  // end of GC_FLAGS
+          constraint(GCCardSizeInBytesConstraintFunc,AtParse)               \
+                                                                            \
+  develop(bool, UseAltGCForwarding, false,                                  \
```

Comment:
If you need this for compact object headers then shouldn't this flag be experimental?

Reply:
@tschatzl requested it to be develop *for this PR* and make it product/experimental in the compact-headers PR, because if the compact-headers PR doesn't make it into 21, then we'd have the slight performance hit of checking the flag (in a loop) in release builds for no benefit. See discussions above. I thought that is reasonable.